extend h5 write test to compare h_loc and perturbation_order

jkarp314 opened the pull request:
I've experienced the problem that when I save and load a CTHYB solver to an h5 file using python, I always get that `solver.h_loc` is 0 and `solver.perturbation_order` is empty. I added these checks to the h5 test

I also fixed comparison of last_solve_parameters

jkarp314 committed:
extend h5 write text to compare h_loc and perturbation_order

also fixed comparison of last_solve_parameters

Wentzell commented:
Dear @jkarp314,

Thank you for pointing this out.
Could you make sure to enable the measurement of the pertubation order in the h5 test
by adding `measure_pert_order: True` to the solver parameters dictionary `sp` in the test.

The storage of `h_loc` was already adressed in 391450537af5f488b6e31433bc72742ef85b9e18,
this should already be fixed in the `2.2.x` branch.

jkarp314 committed:
add measure_pert_order

I added measure_pert_order to the solver parameters and imported pytriqs.statistics.histograms.Histogram

jkarp314 commented:
@Wentzell thanks for the reply

I added `measure_pert_order = True` to the solver parameters.

I also added `from pytriqs.statistics.histograms import Histogram`

Wentzell commented:
This was merged, c.f. commits 1f1a8e9 3d86af8 fbf720c
